.github/workflows/ci.yml

```diff
@@ -126,9 +120,33 @@ jobs:
           CHECK_FORMATTING: '${{ matrix.check_formatting }}'
           # Should match 'name:' up above
           JOB_NAME: 'Ubuntu (${{ matrix.python }}${{ matrix.extra_name }})'
+
+  autofmt:
+    name: Autoformat dependabot PR
+    timeout-minutes: 10
+    if: github.actor == 'dependabot[bot]'
+    runs-on: 'ubuntu-latest'
+    # https://docs.github.com/en/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions#changing-github_token-permissions
+    permissions:
+      pull-requests: write
+      issues: write
+      repository-projects: write
+      contents: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
+      - name: Check formatting
```

Comment:
We can probably combine this with the next check now that it's not the normal check formatting job

Reply:
Ah I see. This is fine then.